packages/clerk-js/src/ui/elements/IconButton.tsx

```diff
@@ -1,21 +1,36 @@
-import React from 'react';
+import { cloneElement, isValidElement } from 'react';
 
 import { Button, Icon, useLocalizations } from '../customizables';
+import { ElementDescriptor } from '../customizables/elementDescriptors';
 import { PropsOfComponent } from '../styledSystem';
 
-export const IconButton = (props: PropsOfComponent<typeof Button> & { icon: React.ComponentType }) => {
-  const { children, icon, localizationKey, ...rest } = props;
+export const IconButton = (
+  props: PropsOfComponent<typeof Button> & {
+    icon: React.ReactElement | React.ComponentType;
+    'aria-label': string;
```

Comment:
Just to confirm, this already exists but we're making this required here right?

Reply:
Not sure I understand. The aria label can be passed to all elements as far as I know, but it's optional. So yes, this is done to enforce the use of it in icon buttons.